README.md

```diff
@@ -2,24 +2,243 @@
+
+#### Proposed modification
+When a specific change to the PMIx Standard has been identified a PR must be opened with a reference made to the corresponding Issue(s). When the PR is opened it should be placed in a [Draft PR](https://github.blog/2019-02-14-introducing-draft-pull-requests/) state until the following conditions are met:
+
+* The contents of the PR are ready for review and broad discussion with the PMIx community. Note that discussion over the PR is encouraged at all times even while in draft form.
+
+* An open-source implementation of the proposal is available and a reference to the implementation included in the PR. Note that a complete implementation is not necessarily required to meet this condition - e.g., pseudocode detailing the execution path supporting the proposed modification can be provided. More complete prototype implementations may be requested by the PMIx community when proposals involving significant changes in behavior are made.
+
+Only new commits are permitted on the branch once the [Draft PR](https://github.blog/2019-02-14-introducing-draft-pull-requests/) designation has been removed. The branch associated with the PR should not be squashed or force pushed after this point to preserve the timeline of changes in relationship to the discussion that may occur on the PR. Note that PRs that are squashed or force pushed remain eligible for consideration, but may increase the difficulty of obtaining consensus from the community.
+
+Discussion of the PR will continue until all dissenting opinions and questions have been addressed, upon which the PR may be put forward for Provisional acceptance. Note that addressing dissenting opinions does not necessarily mean achieving complete resolution on them. A dissent is not considered to have "veto" power over a proposal, but must be addressed to a degree that convinces the majority of ASC members.
+
+#### Provisional acceptance
+The process for acceptance to Provisional status begins when the author(s) assign an "**RFC**" label to the GitHub PR, add an empty "**Straw Poll**" comment to it, and send an announcement of the PR to the PMIx community mailing list requesting consideration of the PR for Provisional status. The announcement shall include a link to the PR, a brief description of the proposed change, and indicate any desired timeline for approval. It is especially important that proposals tied to contractual milestones be so designated to ensure that members are alerted to potential time pressures.
+
+The "Straw Poll" comment is used by the ASC as a mechanism for obtaining a sense of how many people have viewed the PR and are following the discussion. This serves no official purpose other than to help define participation in the consensus building process and record the following tally:
```

Comment:
With the new "Straw Poll" above, maybe we just do a back reference to that paragraph for guidance on the comment to post and the various emojis.

Reply:
Done